Index: koji/builder/kojid
===================================================================
--- koji.orig/builder/kojid
+++ koji/builder/kojid
@@ -52,7 +52,7 @@ from fnmatch import fnmatch
 from gzip import GzipFile
 from optparse import OptionParser, SUPPRESS_HELP
 
-from multilib import multilib
+#from multilib import multilib
 import koji
 import koji.arch
 import koji.plugin
@@ -5572,6 +5572,7 @@ class createDistRepoTask(BaseTaskHandler
 
 
     def do_multilib_dnf(self, arch, ml_arch, conf):
+        raise RuntimeError("multilib is not yet supported when running kojid on Debian, patches welcome")
         repodir = koji.pathinfo.distrepo(self.rinfo['id'], self.rinfo['tag_name'])
         mldir = os.path.join(repodir, koji.canonArch(ml_arch))
         ml_true = set()  # multilib packages we need to include before depsolve
